src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -0,0 +1,432 @@
+				mcreator.getWorkspace().getFolderManager()
+						.getBlockTextureFile(RegistryNameFixer.fix(leavesTextureName)));
+
+		// we use Block GUI to get default values for the block element (kinda hacky!)
+		Block woodBlock = (Block) ModElementTypeRegistry.REGISTRY.get(ModElementType.BLOCK)
+				.getModElement(mcreator, new ModElement(workspace, name + "Wood", ModElementType.BLOCK), false)
+				.getElementFromGUI();
+		woodBlock.name = name + " Wood";
+		woodBlock.material = new Material(workspace, "Wood");
+		woodBlock.texture = woodTextureName;
+		woodBlock.renderType = 11; // single texture
+		woodBlock.customModelName = "Single texture";
+		woodBlock.soundOnStep = new StepSound(workspace, "WOOD");
+		woodBlock.hardness = 2.0 * factor;
+		woodBlock.resistance = 2.0 * Math.pow(factor, 0.8);
+		woodBlock.destroyTool = "axe";
+		woodBlock.breakHarvestLevel = 0;
+		woodBlock.plantsGrowOn = true;
+		woodBlock.flammability = (int) Math.round(5 * factor);
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(woodBlock);
+		mcreator.getWorkspace().addModElement(woodBlock.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(woodBlock);
+		mcreator.getWorkspace().getModElementManager().storeModElement(woodBlock);
+
+		// we use Block GUI to get default values for the block element (kinda hacky!)
+		Block logBlock = (Block) ModElementTypeRegistry.REGISTRY.get(ModElementType.BLOCK)
+				.getModElement(mcreator, new ModElement(workspace, name + "Log", ModElementType.BLOCK), false)
+				.getElementFromGUI();
+		logBlock.name = name + " Log";
+		logBlock.material = new Material(workspace, "Log");
+		logBlock.texture = logTextureName;
+		logBlock.textureTop = logTextureName;
+		logBlock.textureBack = woodTextureName;
+		logBlock.textureFront = woodTextureName;
+		logBlock.textureLeft = woodTextureName;
+		logBlock.textureRight = woodTextureName;
+		logBlock.renderType = 10; // normal
+		logBlock.customModelName = "Normal";
+		logBlock.soundOnStep = new StepSound(workspace, "WOOD");
+		logBlock.hardness = 2.0 * factor;
+		logBlock.resistance = 2.0 * Math.pow(factor, 0.8);
+		logBlock.destroyTool = "axe";
+		logBlock.breakHarvestLevel = 0;
+		logBlock.plantsGrowOn = true;
+		logBlock.flammability = (int) Math.round(5 * factor);
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(logBlock);
+		mcreator.getWorkspace().addModElement(logBlock.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(woodBlock);
+		mcreator.getWorkspace().getModElementManager().storeModElement(logBlock);
+
+		// we use Block GUI to get default values for the block element (kinda hacky!)
+		Block planksBlock = (Block) ModElementTypeRegistry.REGISTRY.get(ModElementType.BLOCK)
+				.getModElement(mcreator, new ModElement(workspace, name + "Planks", ModElementType.BLOCK), false)
+				.getElementFromGUI();
+		planksBlock.name = name + " Planks";
+		planksBlock.material = new Material(workspace, "Wood");

```

Comment:
Material is not set because there isn't a wood material capitalized like that. Please use "WOOD" instead.